src/settings.rs

```diff
@@ -21,6 +21,19 @@ pub const NIX_TARBALL_PATH: &str = env!("NIX_INSTALLER_TARBALL_PATH");
 /// in the resulting binary.
 pub const NIX_TARBALL: &[u8] = include_bytes!(env!("NIX_INSTALLER_TARBALL_PATH"));
 
+#[cfg(all(feature = "determinate-nix", target_os = "linux"))]
+/// The DETERMINATE_NIXD_BINARY_PATH environment variable should point to a target-appropriate
+/// static build of the Determinate Nixd binary. The contents are embedded in the resulting
+/// binary if the determinate-nix feature is turned on.
+pub const DETERMINATE_NIXD_BINARY: Option<&[u8]> =
+    Some(include_bytes!(env!("DETERMINATE_NIXD_BINARY_PATH")));
```

Comment:
You'll need to set this in the `Cargo.toml` to some bogus file, like I did for `NIX_INSTALLER_TARBALL_PATH`, otherwise docs.rs will fail to build our docs because it doesn't have this env var.